### ISSUE TITLE
Custom Skeletons' arrows don't do any damage

### ISSUE THREAD

**Issue by Quasindro** [Syranda/Cardinal#2]
It seems that there's something broken with that. Skeletons shoot normally, but there's no damage on hit. There's also no EntityDamageByEntityEvent called for some reason (at all). I have no idea what could cause this.

On a side note - would you mind if I committed some of my changes/bugfixes to Cardinal? :)

**Comment by Syranda** [comment]
I'll test these bugs when I come home. (Friday)

To your sidenotes: bugfixes yes; changes depend on what you change

**Comment by Quasindro** [comment]
It's Monday now, did you look into this issue?

**Comment by Syranda** [comment]
No. I'm currently very busy with school. I'll look into it when I have time.
